sickbeard/__init__.py

```diff
@@ -1846,7 +1745,7 @@ def save_config():  # pylint: disable=too-many-statements, too-many-branches
     new_config['Blackhole']['torrent_dir'] = TORRENT_DIR
 
     # dynamically save provider settings
-    for curTorrentProvider in [curProvider for curProvider in providers.sortedProviderList() if
+    for curTorrentProvider in [curProvider for curProvider in providers.sort_providers() if
```

Comment:
```AttributeError: 'module' object has no attribute 'sort_providers'```
Forgot to push provider changes? Otherwise this has to stay sortedProviderList.